### ISSUE TITLE
`http: superfluous response.WriteHeader call` (v0.109.0)

### ISSUE THREAD

**Issue by arielvalentin** [open-telemetry/opentelemetry-collector#11239]
v0.109.0 generates an unstructured warning when handling traces: 

```console
otel-collector-1  | 2024/09/21 14:21:15 http: superfluous response.WriteHeader call from go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp/internal/request.(*RespWriterWrapper).writeHeader (resp_writer_wrapper.go:78)
```

Is there a way to disable these warnings?

**Comment by TylerHelmuth** [comment]
It is an error in instrumentation that has been fixed: https://github.com/open-telemetry/opentelemetry-go-contrib/releases/tag/v1.30.0

**Comment by arielvalentin** [comment]
Ok. Will that change be included in v0.110.0?

**Comment by ArquitetoMovel** [comment]
After up from 103.0 to 109.0, I had having several issues like  superfluous response and the zpages extension not works as well. 
I hope to get a new release soon with this fixes.

**Comment by dextercai** [comment]
Will this fix be published with v110?

**Comment by vikrambe** [comment]
When are we planning to cut a release ? Any ETA ?

**Comment by GPegel** [comment]
We are also experiencing the same issue and it is impacting our workflow. A fix for this would be greatly appreciated.

**Comment by arielvalentin** [comment]
I believe the answer is _yes_ the v0.110.0 release will contain a fix for this.

Monolithic repo versioning is a little confusing to a golang novice like myself but here is what I can see. The `v1.30.0` version also includes `v0.55.0` version of the `otelhttp` instrumentation:

https://github.com/open-telemetry/opentelemetry-go-contrib/tree/instrumentation/net/http/otelhttp/v0.55.0/instrumentation/net/http/otelhttp

The `otelcol/main` branch is up to date and has a reference to the v0.55.0 version: https://github.com/open-telemetry/opentelemetry-collector/blob/d17559b6e89a6f97b6800a6538bbf82430d94678/service/go.mod#L97

Assuming all goes well, a release of v0.110.0 is scheduled for 2024-09-23: https://github.com/open-telemetry/opentelemetry-collector/blob/d17559b6e89a6f97b6800a6538bbf82430d94678/docs/release.md#release-schedule

🤞🏼

**Comment by TylerHelmuth** [comment]
A fix was released in v0.110.0 as we are using the updated libraries.

**Comment by arielvalentin** [comment]
Yes it was. The fix is in a transitive dependency
